src/index.js

```diff
@@ -4,23 +4,77 @@
  * Module dependencies.
  */
 
+const { serializeError } = require('serialize-error');
+const cloneDeep = require('lodash/cloneDeep');
```

Comment:
These packages (`lodash/cloneDeep` and `lodash/update`) are not in the dependencies.